client/src/pages/EditProfile/useStyles.ts

```diff
@@ -0,0 +1,52 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
```

Comment:
I don't think we are using this class at the moment? If we plan to, please add it to the relevant component.

Reply:
Resolved